Libraries/ReactNative/AppRegistry.js

```diff
@@ -261,16 +262,22 @@ const AppRegistry = {
     const taskProvider = taskProviders.get(taskKey);
     if (!taskProvider) {
       console.warn(`No task registered for key ${taskKey}`);
-      NativeModules.HeadlessJsTaskSupport.notifyTaskFinished(taskId);
+      if (NativeHeadlessJsTaskSupport) {
```

Comment:
Are these ifs necessary?

Reply:
Flow complains if I don’t. Android only lib. I could attempt to make less of them?

Reply:
Ah ok makes sense